### ISSUE TITLE
How To Use Guide

### ISSUE THREAD

**Issue by TheStophe** [reqlez/tn-ergo-easy-oracle#5]
I've been keeping notes on how to use the docker oracle node, such as:
- extract tokens
- update ergo node version
- update the oracle-core oracle-config.yml to change the explorer to something that works
   - (btw there's no nano command in the `docker exec -ti <ps_number> /bin/sh` area)

Would you like me to create a pull request and create a Daily How To Use guide?

**Comment by reqlez** [comment]
Yes would be nice, however, first I wanted to investigate why people can't just use something like:

docker compose exec -it core /bin/sh -c 'yes YES | oracle-core --oracle-config-file /data/oracle_config.yaml --pool-config-file /data/pool_config.yaml -d /data extract-reward-tokens ADDRESS_TO_EXTRACT_TO_HERE'

Why go into the shell, then change dir etc when you can just run a one-liner?

**Comment by reqlez** [comment]
We can even test and document running this as a cron job to auto-extract every day or week? If you want to test?

docker compose -f /path/to/ergo-easy-oracle/docker-compose.yml exec -it core /bin/sh -c 'yes YES | oracle-core --oracle-config-file /data/oracle_config.yaml --pool-config-file /data/pool_config.yaml -d /data extract-reward-tokens ADDRESS_TO_EXTRACT_TO_HERE'

**Comment by TheStophe** [comment]
Absolutely! It worked. Does it fire off daily at the time of running the script?

**Comment by reqlez** [comment]
> Absolutely! It worked. Does it fire off daily at the time of running the script?

if you set it in your crontab yes u can do daily if you want.
